ipfs-pinning-service.yaml

```diff
@@ -74,12 +74,10 @@ issues in restrictive network topologies such as NAT.
 
 Pinning services are encouraged to add support for additional features by leveraging the following optional `meta` attributes. Note that it is OK to ommit or ignore `meta` attributes; doing so should not impact the basic pinning functionality.
 
-- `PinStatus.meta[progress]`: Progress (as percent value) of an ongoing pinning operation, if possible to tell
 
-- `PinStatus.meta[fail_reason]`: A service-specific reason why a pin operation failed (e.g. lack of funds, DAG too big, etc.)
-
-- `Pin.meta[replication]`: This attribute could define how many copies a pinning service should keep
+- `Pin.meta[app_id]`: Attaching unique identifier to pins created by an app enables filtering pins per app via `?meta={\"app_id\":<UUID>}`
 
+- `PinStatus.meta[status_details]`: A service-specific details. For example, when `PinStatus.status=failed` it could provide a reason why a pin operation failed (e.g. lack of funds, DAG too big, etc.)
```

Comment:
```suggestion
- `PinStatus.meta[status_details]`: Service-specific details. For example, when `PinStatus.status=failed` it could provide a reason why a pin operation failed (e.g. lack of funds, DAG too big, etc.)
```